### DISCUSSION TITLE
Increase editor area and saving node with custom arguments

### DISCUSSION THREAD

**Discussion by yegorf** [eckig/graph-editor#186] [Q&A] [answered]
Hello Steffen, thanks for your work.

I have 2 questions:
1) How to increase the size of the editor area (or make it endless).
![Screenshot 2024-04-17 102442](https://github.com/eckig/graph-editor/assets/29003239/30bd7734-9b98-435b-a0c3-3e8d76f4e2be)
2) How to save the Nodes and Connectors with my custom arguments?
![image](https://github.com/eckig/graph-editor/assets/29003239/259cdadb-2db8-41ed-a350-8ad8236dd382)

Thank you.

**Comment by eckig** [comment, marked as answer]
1. Endless canvas size is currently not supported. But you can adjust the size with the `contentHeight` and `contentWidth` attributes of the `GModel`.
2. If you have any custom attributes, you should store them in a separate [domain model](https://github.com/eckig/graph-editor/wiki/4.-Connecting-to-Domain-Models). Then you can save both models to the desired output format.

**Reply by yegorf** [reply]
Thanks a lot for the quick reply, I'll try these solutions.
